ext/proj4_c_impl/extconf.rb

```diff
@@ -45,11 +45,13 @@
 
   found_proj_ = false
   header_dirs_, lib_dirs_ = dir_config("proj", header_dirs_, lib_dirs_)
-  dflag = "-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H"
-  if have_header("proj_api.h", nil, dflag)
+  if have_header("proj.h")
     $libs << " -lproj"
-    if have_func("pj_init_plus", "proj_api.h", dflag)
+
+    if have_func("proj_create", "proj.h")
       found_proj_ = true
+      have_func("proj_create_crs_to_crs_from_pj", "proj.h")
+      have_func("proj_normalize_for_visualization", "proj.h")
```

Comment:
Yeah I know it won't block, but you're right I think we should try to find docs or look at other gems. Seems that good documentation on this stuff is kind of lacking so it might require some digging.